.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
I would remove all other php versions beside 7.0. We are only running the phpcs build so running it on multiple php versions is pointless.